src/engine/sys/sdl_glimp.cpp

```diff
@@ -966,7 +966,8 @@ static void RequireExt( bool hasExt, const char* name )
 	}
 	else
 	{
-		logger.WithoutSuppression().Notice("...GL_%s not found", name );
+		SDL_DestroyWindow( window );
```

Comment:
This will be done already by `Sys::Error -> Sys::Shutdown -> Application::Shutdown -> ClientApplication::Shutdown -> CL_Shutdown -> CL_ShutdownRef -> re.Shutdown = RE_Shutdown -> GLimp_Shutdown -> SDL_DestroyWindow`

Looks good otherwise.

Reply:
Yeah, I was wondering about the fact the window is destroyed after the error message is displayed, leaving undefined content on screen until the user acknowledges the error.

Reply:
We may modify `Sys::Error` to destroy the window before printing the error message in another PR instead.